uacp/conn.go

```diff
@@ -367,7 +367,10 @@ func (c *Conn) Receive() ([]byte, error) {
 	}
 
 	if h.MessageSize > c.ack.ReceiveBufSize {
-		return nil, errors.Errorf("uacp: message too large: %d > %d bytes", h.MessageSize, c.ack.ReceiveBufSize)
+		return nil, errors.Errorf("uacp: message too large: %d > %d bytes. MsgType=%s, ChunkType=%c", h.MessageSize, c.ack.ReceiveBufSize)
```

Comment:
Seems like there are some missing fmt params here.

Reply:
Sorry about that - fixed.